.github/dependabot.yaml

```diff
@@ -1,5 +1,13 @@
 version: 2
 updates:
+  - package-ecosystem: "gomod"
```

Comment:
I'm not sure how gno will be updated as a dependency, since we don't do releases yet 🤔

Reply:
didn't know it was not working on pseudo-release, I will remove dependabot for now